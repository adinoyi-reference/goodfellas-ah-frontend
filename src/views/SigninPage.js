import React from 'react';

import Auth from '../components/layout/Auth';
import SigninForm from '../containers/SigninForm';

const Signin = (props) => (
  <div>
    <Auth>
      <SigninForm {...props} />
    </Auth>
  </div>
);

export default Signin;
