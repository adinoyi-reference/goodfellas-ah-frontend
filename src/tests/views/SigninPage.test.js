import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SigninPage from '../../views/SigninPage';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<SigninPage />);
});


describe('Signin UI', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});
