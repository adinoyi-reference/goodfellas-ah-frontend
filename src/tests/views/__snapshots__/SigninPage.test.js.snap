// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SigninPage UI render features view should render as expected 1`] = `
<div>
  <Auth>
    <SigninForm />
  </Auth>
</div>
`;
